### PR TITLE
README.md: add use case for container introspection

### PR DESCRIPTION
**Pull request by vbatts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ distribution specific.
 
 If you would like to see additional names and desciptions added please open issues and send pull requests to update this readme.
 
-We are looking at two types of data here. 
+The follwing types of data are being considered:
```
Comment (jonboulle) [inline README.md:14]: following

File: README.md [hunk 2]
```diff
@@ -37,3 +37,8 @@ We are looking at two types of data here.
  | Summary     | Short Description of the image|
  | Description | Detailed discription of the image|
 
+3. Annotations with container configuration that is externally accesible, to clarify the extent to which a container image is a "black box". Rather than explicit fields, this is a pattern for the key (in likeness to an Object Identifiers (OID) or the Java package naming convention).
```
Comment (jonboulle) [inline README.md:40]: accessible

**Comment by rhatdan** [comment]
Also fixed several spelling mistakes. Mine and @vbatts .

**Pull request merged**